src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Write.String.cs

```diff
@@ -6,11 +6,32 @@ namespace System.Text.Json.Serialization
 {
     public static partial class JsonSerializer
     {
+        /// <summary>
+        /// Convert the value and return as <see cref="System.String"/>.
+        /// </summary>
+        /// <returns>The converted value.</returns>
+        /// <param name="value">The value to convert.</param>
+        /// <param name="options">The options used to convert the value.</param>
+        /// <remarks>Using a UTF-16 <see cref="System.String"/> is not as efficient as a
```

Comment:
Maybe call-out (as a `seealso`) the ToBytes method here?

Reply:
Done